roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -27,9 +110,108 @@ No initialization routine runs for this module.
 
 ## Routines
 
+The following routines are intended to be invoked by paras' upward messages.
+
+* `init_open_channel(recipient)`:
```

Comment:
rename to `hrmp_*` for these functions? 

also, as an architecture question, might it make sense to have a separate `Hrmp` module? Or is everything in `Router` module OK?

Reply:
When I was prefixing everything (and apparently missed these messages) I was tempted to extract this into a separate module. But then I got thinking that then XCMP will need to be placed in a separate module only leaving VMP here which kinda ruins the whole picture. And I haven't. So this is something to think about.